CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 - Deployment Plan
 - Enclosure
 - Enclosure Group
+- Hypervisor Cluster Profile
```

Comment:
Just a suggestion here: To add new resources, lets add it under #New Resources heading.

Reply:
Added in Notes .